index.html

```diff
@@ -97,5 +97,196 @@ <h1 class="primary-text">Hey There. I'm Alan</h1>
+                </svg>
+                <hr class="separator">
+                <div class="about-me-features">
+                    <div class="feature">
+                        <div class="feature-title">
+                            <svg width="27" height="26" viewBox="0 0 27 26" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <rect x="0.772461" y="12.7279" width="18" height="18" transform="rotate(-45 0.772461 12.7279)" fill="#FF6B00"/>
+                            </svg>
+                            <h2 class="feature-title-heading white-font">Language</h2>
+                        </div>
+                        <ul class="white-font">
+                            <li class="feature-items">Javascript</li>
+                            <li class="feature-items">Ruby</li>
+                            <li class="feature-items">Html</li>
+                            <li class="feature-items">Css</li>
+                        </ul>
+                    </div> 
+                    <div class="feature">
+                        <div class="feature-title">
+                            <svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <rect y="0.455811" width="18" height="18" fill="#FF6B00"/>
+                            </svg>
+                            <h2 class="feature-title-heading white-font">Frameworks</h2>
+                        </div>
+                        <ul class="white-font">
+                            <li class="feature-items">Bootstrap</li>
+                            <li class="feature-items">Ruby on Rails</li>
+                            <li class="feature-items">RSPes</li>
+                            <li class="feature-items">CapyBara</li>
+                            <li class="feature-items">Selenium</li>
+                        </ul>
+                    </div>
+                    <div class="feature">
+                        <div class="feature-title">
+                            <svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <rect y="0.455811" width="18" height="18" rx="9" fill="#FF6B00"/>
+                            </svg>
+                            <h2 class="feature-title-heading white-font">Skills</h2>
+                        </div>
+                        <ul class="white-font">
+                           <li class="feature-items">Coedkit</li>
+                           <li class="feature-items">Github</li>
+                           <li class="feature-items">Codepen</li>
+                           <li class="feature-items">Gitlab</li>
+                           <li class="feature-items">Terminal</li>
+                       </ul>
+                   </div>
+                </div>
+        </div>
```

Comment:
- [ ] It's required to have the container of the about me section in its own `section` tag outside of the work section.